lib/util/Ordering.ts

```diff
@@ -2,10 +2,14 @@ import type * as RDF from '@rdfjs/types';
 import * as LRUCache from 'lru-cache';
 import type { LangStringLiteral } from '../expressions';
 import { TermTransformer } from '../transformers/TermTransformer';
-import type { MainSparqlType } from './Consts';
-import type { ISuperTypeProvider, SuperTypeCallback, TypeCache } from './TypeHandling';
+import { TypeAlias, TypeURL } from './Consts';
+import type { ISuperTypeProvider, SuperTypeCallback, TypeCache, GeneralSuperTypeDict } from './TypeHandling';
+import { getSuperTypeDict } from './TypeHandling';
 
 // Determine the relative numerical order of the two given terms.
+/**
+ * @param enableExtendedXSDTypes deprecated since version ... . System will behave like when this was true.
```

Comment:
```suggestion
 * @param enableExtendedXSDTypes System will behave like when this was true. @deprecated
```

Reply:
Seems like it doesn't technically do anything to add the `@deprecated` but at least in this case it is indicated (be it in string format).